IdentityCore/src/oauth2/token/MSIDPrimaryRefreshToken.m

```diff
@@ -85,6 +87,8 @@ - (MSIDCredentialCacheItem *)tokenCacheItem
     prtCacheItem.expiresOn = self.expiresOn;
     prtCacheItem.cachedAt = self.cachedAt;
     prtCacheItem.lastRecoveryAttempt = self.lastRecoveryAttempt;
+    prtCacheItem.recoveryAttemptCount = [NSString stringWithFormat:@"%lu", (long)self.recoveryAttemptCount];
```

Comment:
Any unit tests to update for this conversion?

Reply:
added extra tests